lib/experimental/class-wp-theme-json-gutenberg.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * WP_Theme_JSON_Gutenberg class
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that encapsulates the processing of structures that adhere to the theme.json spec.
+ *
+ * This class is for internal core usage and is not supposed to be used by extenders (plugins and/or themes).
+ * This is a low-level API that may need to do breaking changes. Please,
+ * use get_global_settings, get_global_styles, and get_global_stylesheet instead.
+ *
+ * @access private
+ */
+class WP_Theme_JSON_Gutenberg extends WP_Theme_JSON_6_1 {
```

Comment:
Why do we need this here instead of naming `WP_Theme_JSON_Gutenberg` the class in `6.1`? Is there a specific reason?

Reply:
The reason is that when we come to add more compat versions of this file we need to change the class name to the specific version and add a new one to the next compat folder which can create conflicts. Keeping the `WP_Theme_JSON_Gutenberg` in experimental is more stable and predictable.

Reply:
I guess it's how things are done for this particular class that gets updated in every WordPress major release.